authzed/api/v1/experimental_service.proto

```diff
@@ -44,19 +44,67 @@ service ExperimentalService {
       };
     }
 
+  // NOTE: BulkCheckPermission has been promoted to the stable API as "CheckBulkPermission" and the
+  // API will be removed from experimental in a future release.
   rpc BulkCheckPermission(BulkCheckPermissionRequest)
     returns (BulkCheckPermissionResponse) {
       option (google.api.http) = {
         post: "/v1/experimental/permissions/bulkcheckpermission"
         body: "*"
       };
+      option deprecated = true;
+    }
+
+  // EXPERIMENTAL: ReflectSchema is an API that allows clients to reflect the schema stored in
+  // SpiceDB. This is useful for clients that need to introspect the schema of a SpiceDB instance.
+  rpc ExperimentalReflectSchema(ExperimentalReflectSchemaRequest)
+    returns (ExperimentalReflectSchemaResponse) {
+      option (google.api.http) = {
+        post: "/v1/experimental/reflectschema"
+        body: "*"
+      };
+    }
+
+  // EXPERIMENTAL: ComputablePermissions is an API that allows clients to request the set of
+  // permissions that compute based off a set of relations. For example, if a schema has a relation
+  // `viewer` and a permission `view` defined as `permission view = viewer + editor`, then the
+  // computable permissions for the relation `viewer` will include `view`.
+  rpc ExperimentalComputablePermissions(ExperimentalComputablePermissionsRequest)
+    returns (ExperimentalComputablePermissionsResponse) {
+      option (google.api.http) = {
+        post: "/v1/experimental/permissions/computable"
+        body: "*"
+      };
+    }
+  
+  // EXPERIMENTAL: DependentRelations is an API that allows clients to request the set of
+  // relations that used to compute a permission, recursively. It is the inverse of the
+  // ComputablePermissions API.
+  rpc ExperimentalDependentRelations(ExperimentalDependentRelationsRequest)
+    returns (ExperimentalDependentRelationsResponse) {
+      option (google.api.http) = {
+        post: "/v1/experimental/permissions/dependent"
+        body: "*"
+      };
+    }
+
+  // EXPERIMENTAL: DiffSchema is an API that allows clients to request the difference between the
+  // specified schema and the schema stored in SpiceDB. This is useful for clients that need to
+  // introspect the schema of a SpiceDB instance.
+  rpc ExperimentalSchemaDiff(ExperimentalSchemaDiffRequest)
```

Comment:
This wasn't in the original proposal - what's the use-case you were imagining?

Reply:
https://github.com/authzed/spicedb/issues/1804